src/uu/expr/src/syntax_tree.rs

```diff
@@ -79,24 +80,18 @@ impl RelationOp {
             }
         };
         if b {
-            Ok("1".into())
+            Ok(NumOrStr::from(1))
```

Comment:
```suggestion
            Ok(1.into())
```
I suggested the `From` implementation to make this possible, otherwise it's not really much more convenient than `NumOrStr::Num` and `NumOrStr::Str` :)

Reply:
Yes, makes sense. Updated all the `from` usages.